Sources/MongoSwift/MongoCollection+Write.swift

```diff
@@ -31,7 +31,11 @@ extension MongoCollection {
             return nil
         }
 
-        return InsertOneResult(insertedId: try document.getValue(for: "_id"))
+        if let insertedId = try document.getValue(for: "_id") {
```

Comment:
nit: I think
```swift
guard let insertedId = ... else { 
   // throw 
}}

 return InsertOneResult(insertedId: insertedId)
```
is more swifty if we're gonna throw in the `else`

Reply:
You are totally right. Guess I missed some spots.

I realized that it will be weird when I move onto a language without `guard`. Once I form the habit of using `guard`'s like this, I'll really miss it in other languages.

Done.